decidim-participatory_processes/app/commands/decidim/participatory_processes/admin/update_participatory_process_admin.rb

```diff
@@ -5,15 +5,16 @@ module ParticipatoryProcesses
     module Admin
       # A command with all the business logic when updated a participatory
       # process admin in the system.
-      class UpdateParticipatoryProcessAdmin < Rectify::Command
+      class UpdateParticipatoryProcessAdmin < NotifyRoleAssignedParticipatoryProcess
         # Public: Initializes the command.
         #
         # form - A form object with the params.
         # participatory_process - The ParticipatoryProcess that will hold the
         #   user role
-        def initialize(form, user_role)
+        def initialize(form, user_role, participatory_process)
           @form = form
           @user_role = user_role
+          @participatory_process = participatory_process
```

Comment:
as commented in previous commands:

```suggestion
          @participatory_process = form.current_participatory_space
```